widgets/Salesforce-batchsink.json

```diff
@@ -232,6 +273,23 @@
           "name": "connection"
         }
       ]
+    },
+    {
+      "name": "Proxy authentication",
+      "condition": {
+        "property": "proxyUrl",
+        "operator": "exists"
+      },
+      "show": [
```

Comment:
Does this show up only on providing proxy URL ?

Reply:
yes, below will be shown only after user entered something in this field.